modules/private/key.go

```diff
@@ -1,131 +1,19 @@
-// Copyright 2018 The Gitea Authors. All rights reserved.
+// Copyright 2019 The Gitea Authors. All rights reserved.
```

Comment:
This should not be changed. It will always put the year the file created.

Reply:
```suggestion
// Copyright 2018 The Gitea Authors. All rights reserved.
```